src/index.js

```diff
@@ -15,7 +15,11 @@ export default class HyperWebView extends Component {
     });
   }
 }
-const HyperWebViewManager = requireNativeComponent('HyperWebViewManager');
+
+const HyperWebViewManager =
+  Platform.OS === 'ios'
```

Comment:
@balaganesh-juspay Can we not name them the same? I'm guessing that only one would be native component would be loaded at any one point.

Reply:
@ShreyBana It was giving not found error when I used the same name. That's why changed it. But not sure why.